client/src/components/App.jsx

```diff
@@ -18,9 +20,26 @@ export default class App extends React.Component {
     });
   }
 
+  showModal() {
+    this.setState({
+      modalDisplay: 'block'
+    });
+  }
+
+  hideModal() {
+    this.setState({
+      modalDisplay: 'none'
+    });
+  }
+

```

Comment:
Did you consider using a toggleModal() function that covered the functionality of both showModal() and hideModal()?